web/packages/extension/assets/options.html

```diff
@@ -29,6 +29,10 @@
                 <input type="checkbox" id="autostart" />
                 <label for="autostart">Autoplay Flash content (click to unmute)</label>
             </div>
+            <div class="option checkbox">
+                <input type="checkbox" id="swf_takeover" />
+                <label for="swf_takeover">Redirect SWF links to SWF Player</label>
```

Comment:
I am not sure if this is really clear. Opening a page is not really a link? Maybe "Play SWF files in browser instead of downloading" or something?

Reply:
Ok, this works, I was slightly worried it was going to be really long and take up too much space, but at least in English, it's still just two lines, so I think it's fine.
![Screenshot 2024-06-30 105005](https://github.com/ruffle-rs/ruffle/assets/29206584/a537473d-b7e7-456f-9ef8-7a9d5784ef76)